apps/funding-platform/src/blockchain/index.ts

```diff
@@ -0,0 +1,93 @@
+import { ethers, providers, Signer, Wallet } from "ethers";
+import { hardhatChainId } from "../utils";
+
+const productionChainIds: [number, ethers.utils.ConnectionInfo][] = [
+  [100, { url: "https://rpc.gnosischain.com/" }],
+];
+const developmentChainIds: [number, ethers.utils.ConnectionInfo][] = [
+  [hardhatChainId, { url: "http://localhost:8545" }],
+];
+export const chainIds = new Map<number, ethers.utils.ConnectionInfo>(
```

Comment:
I would rather pass the `chainIds` into the function that requires them after they have been initialized in `index.ts`
this clears up the dependency tree